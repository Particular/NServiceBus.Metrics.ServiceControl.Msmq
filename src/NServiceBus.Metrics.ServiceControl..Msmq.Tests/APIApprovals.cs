using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(MsmqExtensions).Assembly);
        Approver.Verify(publicApi);
    }
}
