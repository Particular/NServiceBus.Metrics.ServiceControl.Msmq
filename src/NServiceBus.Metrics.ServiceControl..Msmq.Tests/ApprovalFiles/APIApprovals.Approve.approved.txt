[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("NServiceBus.Metrics.ServiceControl.Msmq.Tests")]
[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]